pkg/client/unversioned/jobs.go

```diff
@@ -85,7 +89,7 @@ func (c *jobs) Delete(name string, options *api.DeleteOptions) (err error) {
 		return c.r.Delete().Namespace(c.ns).Resource("jobs").Name(name).Do().Error()
 	}
 
-	body, err := api.Scheme.EncodeToVersion(options, c.r.APIVersion())
+	body, err := api.Scheme.EncodeToVersion(options, latest.GroupOrDie("").GroupVersion)
```

Comment:
Horizontal Pod Autoscaler is also experimental.

https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/horizontalpodautoscaler.go#L76

@caesarxuchao can you advise?